qemu/qemu.go

```diff
@@ -863,7 +876,11 @@ func (scsiCon SCSIController) QemuParams(config *Config) []string {
 	var qemuParams []string
 	var devParams []string
 
-	devParams = append(devParams, fmt.Sprintf("virtio-scsi-pci,id=%s", scsiCon.ID))
+	if runtime.GOARCH == "s390x" {
```

Comment:
now I think this condition is not required, since `VirtioSCSI` changes its values at `init()` if `runtime.GOARCH == "s390x"`, right?

Reply:
sorry oversight changes done